src/posthog-featureflags.ts

```diff
@@ -279,7 +279,16 @@ export class PostHogFeatureFlags {
         if (this.instance.decideEndpointWasHit) {
             const flags = this.getFlags()
             const flagVariants = this.getFlagVariants()
-            callback(flags, flagVariants)
+
```

Comment:
the `featureFlagsHandlers` expect a similar schema, so you'll need to update `receivedFeatureFlags` as well. Worth creating this into a function like `prepareFeatureFlagsForCallbacks`

Reply:
and corresponding test too 😅